llvm/lib/Analysis/ValueTracking.cpp

```diff
@@ -6947,6 +6947,81 @@ bool llvm::onlyUsedByLifetimeMarkersOrDroppableInsts(const Value *V) {
       V, /* AllowLifetime */ true, /* AllowDroppable */ true);
 }
 
+bool llvm::isLanewiseOperation(const Instruction *I) {
+  if (auto *II = dyn_cast<IntrinsicInst>(I)) {
```

Comment:
Please do not extend the list of intrinsics as part of this change. We'll probably want to deduplicate this with isTriviallyVectorizable.

Reply:
Can we just check `isTriviallyVectorizable` instead of this match, as a follow-up?

Reply:
Yes, I think so.